docs/src/install-guide.md

```diff
@@ -14,19 +14,20 @@ GitHub CI workflows, see [GitHub CI Action](./install-github-ci.md).
 
 The following must already be installed:
 
-* **Python version 3.7 or newer** and the package installer `pip`.
 * Rust 1.58 or newer installed via `rustup`.
 
 ## Installing the latest version
 
 To install the latest version of Kani, run:
 
 ```bash
-cargo install --locked kani-verifier
-cargo kani setup
+cargo install --locked kani-verifier # build Kani's installer
+cargo kani setup # run the installer to download and install the prebuilt binaries as well as supporting libraries and data
```

Comment:
May I suggest to break this pre-formatted code block into two and have the comments as plain text instead? Especially the second comment is very long, and pre-formatting will require that it be rendered as a single line.

Reply:
Done. Let me know if this looks better.